src/app/static-analysis.js

```diff
@@ -0,0 +1,54 @@
+module.exports = { analyze: analyze };
+
+// source: just a big string of the current tab's source code.
+//
+// compiledResult: an object with different keys based on success/failure of compilation.
+//   success keys: 'contracts' (contains opcodes, bytecode) and 'sources' (contains AST)
+//   error key: 'error' (contains compile error string)
+
+function analyze (source, compiledResult) {
+  var messages = [];
+
+  if (compiledResult) {
+    if (compiledResult.contracts) {
+      Object.keys(compiledResult.contracts).forEach(function (contractName) {
+        // @TODO: use compiledResult.contracts[contractName].opcodes, interface, etc
+      });
+    }
+    if (compiledResult.sources) {
+      Object.keys(compiledResult.sources).forEach(function (tabName) {
+        var ast = compiledResult.sources[tabName].AST;
+        if (searchAST(ast, {member_name: 'send'})) {
+          messages.push('Caution: You are using the send method.\nPlease read <a target="_blank" href="http://hackingdistributed.com/2016/06/16/scanning-live-ethereum-contracts-for-bugs/">this</a> before deploying your code.');
```

Comment:
Unchecked send is already reported by the compiler.

Furthermore, a core principle of the compiler is that any error and warning can be silenced by the user in a certain way (either by correcting the error or adding a certain pattern that says "yes, I know what I am doing").
Adding a warning for every single send will just cause the users to ignore all warnings, even the serious ones.

I would suggest creating a new "pane" where the user can see results of static analysis, with checkboxes that allow certain analyses to be performed and an explanation what the analysis is for.